### PR TITLE
Improving the wording of human hails

### PR DESCRIPTION
**Pull request by bene-dictator**
**Content (Hails)**

## Summary
~Fixes a new Navy hail~ and changes some other ones (Militia, Pirate) to have slightly more variation. 

Since #8043 did what I meant to do in this PR but got merged first, this PR will just be a pass over for some new hails.

### CODE REVIEW

**Comment by Zitchas** [comment]
Just a general suggestion for PR naming: Please include a bit more detail. For instance, "whose hails are you fixing?" This just helps us know what to expect when we get into it and who needs to be paying particular attention.

File: data/human/hails.txt [hunk 1]
```diff
@@ -3204,13 +3211,14 @@ phrase "friendly militia 4 variation"
...
 		" what "
 	word
-		"pirates"
-		"criminals"
-		"warlords"
-		"marauders"
-		"rampant piracy"
+		"pirates have"
+		"criminals have"
+		"warlords haves"
```
Comment (Ferociousfeind) [inline data/human/hails.txt:3216]: _SCREEEEEEEEEEEE_

Comment (bene-dictator) [reply data/human/hails.txt:3216]: I'm not having another PR take away from this one so easily!

File: data/human/hails.txt [hunk 2]
```diff
@@ -3112,17 +3112,23 @@ phrase "friendly militia 3"
...
+		"those representatives of Parliament"
 	word
 		" would "
-		" would come down here and "
-		" would stop hiding up in the Paradise Worlds and "
-		" would stop licking the boots of the Syndicate and "
+	word
+		""
+		"come down here and "
+		"stop hiding up in the Paradise Worlds and "
```
Comment (Saugia) [inline data/human/hails.txt:3123]: I feel the "up" in this hail is unnecessary. What is it referring to? Even if whatever fleet that uses this hail is always "below" Paradise worlds, it's still an implication.

That said, "the" could also be taken out. Saying "the Paradise Worlds" is basically the same thing as saying "Paradise Worlds". Leaving:

".....stop hiding in paradise worlds and......"

**Comment by Saugia** [comment]
Took back my comment. On second thought it made sense for the Militia.

**Comment by bene-dictator** [comment]
I could change it to "come down here".

**Review by roadrunner56 — APPROVED** [review]

**Pull request merged**